src/frontend/src/optimizer/plan_node/batch_log_seq_scan.rs

```diff
@@ -126,6 +129,8 @@ impl TryToBatchPb for BatchLogSeqScan {
                     },
                 )),
             }),
+            // It's currently true.
+            ordered: true,
```

Comment:
Let's make this consistent with [BatchSeqScan](https://github.com/risingwavelabs/risingwave/blob/45b7b2a389edcfc331a8d6b8fea24645d5eb3cb2/src/frontend/src/optimizer/plan_node/batch_seq_scan.rs#L253):
```
ordered: !self.order().is_any()
```